internal/zero/cmd/env.go

```diff
@@ -50,3 +58,26 @@ func getOTELAPIEndpoint() string {
 	}
 	return "https://telemetry.pomerium.app"
 }
+
+func getBootstrapConfigFileName() (string, error) {
+	if filename := os.Getenv(BootstrapConfigFileName); filename != "" {
+		return filename, nil
+	}
+	cacheDir, err := os.UserCacheDir()
+	if err != nil {
+		return "", err
+	}
+	dir := filepath.Join(cacheDir, "pomerium")
+	if err := os.MkdirAll(dir, 0o700); err != nil {
+		return "", fmt.Errorf("error creating cache directory: %w", err)
+	}
+
+	return filepath.Join(dir, "bootstrap.dat"), nil
+}
+
+func getBootstrapConfigWritebackURI() string {
+	if uri := os.Getenv(BootstrapConfigWritebackURI); uri != "" {
+		return uri
+	}
+	return ""
```

Comment:
```suggestion
	return os.Getenv(BootstrapConfigWritebackURI)
```

Reply:
This is an intentional pattern, but not a big deal, I will change it. :+1: